Gemfile

```diff
@@ -5,7 +5,7 @@ gem "mime-types", "~> 2.99", :platforms => [:ruby_19, :jruby]
 gem "bump"
 gem "rake"
 gem "rspec"
-gem "rubocop", "0.39.0"
+gem "rubocop", ">= 0.49.0"
```

Comment:
this should be `~> 0.64.0` so it does not randomly update and breaks

Reply:
Updated.